MobileWallet/Screens/Send/AddRecipient/AddRecipientViewController.swift

```diff
@@ -67,14 +67,16 @@ class AddRecipientViewController: UIViewController, UITextFieldDelegate, Contact
 
     private var selectedRecipientPublicKey: PublicKey? = nil {
         didSet {
-            if selectedRecipientPublicKey != nil {
+            if let _ = selectedRecipientPublicKey {
                 inputBox.textAlignment = .center
                 inputBox.returnKeyType = .continue
                 UIImpactFeedbackGenerator(style: .light).impactOccurred()
-                contactsTableVC.tableView.removeFromSuperview()
+                contactsTableVC.tableView.isHidden = true
```

Comment:
@Jasonvdb Not sure if I'm reading this right _(I wasn't able to run the code - updated my clone but still having issues, will write to you about it tomorrow)_ but Android and iOS might be diverging here a bit. Android shows the list and makes a search regardless of the input state: text, pasted emoji id or partial emoji id. Ignore please if not relevant.

Reply:
Hmm yeah so it looked to me from the designs that once they've tapped a contact, pasted an emoji ID or scanned a QR that the contacts list should disappear: https://zpl.io/adGoXyp